samples/tfm/tfm_secure_peripheral/README.rst

```diff
@@ -70,9 +68,24 @@ To start using a peripheral as a secure peripheral, complete the following steps
 
       CONFIG_NRF_GPIO0_PIN_MASK_SECURE=0x00800000
 
-#. Assign the peripheral to the specific partition in the partition manifest YAML file `tfm_secure_peripheral_partition.yaml`_, located in the :file:`secure_peripheral_partition` directory:
+#. Assign the peripheral to the specific partition in the partition manifest YAML file `tfm_secure_peripheral_partition.yaml`_, located in the :file:`secure_peripheral_partition` directory.
+   This step is required when the peripheral has security attributes that can be set by the user (such as split security attributes).
+   Assigning the peripheral makes sure that the TF-M configures this peripheral as secure.
+
+   For example, the following MMIO definition configures the TIMER1 peripheral as secure:
+
 
    .. code-block:: yaml
+   
+   
+      "mmio_regions": [
+              {
+                      "name": "TFM_PERIPHERAL_TIMER1",
+                      "permission": "READ-WRITE"
+              },
+      ]
+   
+   See `TF-M Secure Interrupt Integration`_  for more details on the MMIO regions.
 
       "mmio_regions": [
```

Comment:
You haven't removed this "old" code blocks from before the edits. They are now duplicated, and these old lines are breaking the build. Remove lines 90-95.

Reply:
Removed both, thanks!